app/src/main/java/com/example/cm/data/models/MeetupRequest.java

```diff
@@ -51,12 +64,37 @@ public void setLocation(String location) {
         this.location = location;
     }
 
-    public String getMeetupAt() {
+    public Date getMeetupAt() {
         return meetupAt;
     }
 
-    public void setMeetupAt(String meetupAt) {
+    public void setMeetupAt(Date meetupAt) {
         this.meetupAt = meetupAt;
+        calendarMeetup.setTime(meetupAt);
+    }
+
+    @SuppressLint("DefaultLocale")
+    @Exclude
+    public String getTimeDisplayed(){
```

Comment:
Same here, maybe rename to getFormattedTime?

Reply:
Done